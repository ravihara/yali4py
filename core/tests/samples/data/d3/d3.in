d3 - sample file with .in extension