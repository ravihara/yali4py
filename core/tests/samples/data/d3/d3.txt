d3 - sample file with .txt extension