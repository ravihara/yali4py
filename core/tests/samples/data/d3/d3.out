d3 - sample file with .out extension